e2e-polybft/cardanofw/test-apex-system.go

```diff
@@ -209,7 +209,17 @@ func (a *ApexSystem) InitContracts(ctx context.Context) error {
 
 func (a *ApexSystem) FundWallets(ctx context.Context) error {
 	return a.execForEachChain(func(chain ITestApexChain) error {
-		return chain.FundWallets(ctx)
+		return chain.FundWallets(ctx, big.NewInt(0))
+	})
+}
+
+func (a *ApexSystem) FundChainWallets(ctx context.Context, chainID string, fundAmount *big.Int) error {
+	return a.execForEachChain(func(chain ITestApexChain) error {
```

Comment:
use `GetChainMust`

Reply:
actually this function is kind of confusing... it funds all but `chainID` chain

Reply:
this was last minute change and I made a bug, it should only fund chain that matches chainID